webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,17 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand",function(){})
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.style.overflow = 'hidden';
+        this.style.height = 0;
```

Comment:
Once we delete the long comment, the height should decrease too. right? Probably, for that we will need to set height =0